mathesar_ui/README.md

```diff
@@ -71,15 +71,24 @@ For guidelines on component development, refer [README of components](https://gi
 You can start storybook in dev mode by connecting to the container and then running `npm run storybook`:
 
 ```bash
-docker exec --user mathesar -it mathesar_service_1 /bin/bash
+docker exec -it mathesar_service_1 /bin/bash
 
-node@c273da65c52d:/code/mathesar_ui$ cd mathesar_ui && npm run storybook
+root@c273da65c52d:/code/mathesar_ui$ cd mathesar_ui && npm run storybook
 ```
 
 This should start storybook at port 6006.
 
 To build storybook, run `npm run build-storybook`. The static storybook build will be present within __storybook-static__ folder.
 
+### Developing in Windows
+Hot module replacement does not work well with WSL, as mentioned here as mentioned in [this issue](https://github.com/microsoft/WSL/issues/4739), when the project is present within a windows filesystem.
+
+The simplest way to get this work, and the adviced option, is to move the project to a linux filesystem. You can clone the repo within the WSL shell in a linux filesystem, which should resolve this.
```

Comment:
```suggestion
The simplest way to get this to work, and the advised option, is to move the project to a linux filesystem. You can clone the repo within the WSL shell in a linux filesystem, which should resolve this.
```

Reply:
```suggestion
The simplest way to get this to work (and the one we advise) is to move the project to a Linux filesystem. This can be achieved by cloning the repo within the WSL shell in a Linux filesystem.
```